abi/contracts/Baal.sol/BaalSummoner.json

```diff
@@ -0,0 +1,293 @@
+[
+  {
+    "inputs": [
+      {
+        "internalType": "address payable",
+        "name": "_template",
+        "type": "address"
+      },
+      {
+        "internalType": "address",
+        "name": "_gnosisSingleton",
+        "type": "address"
+      },
+      {
+        "internalType": "address",
+        "name": "_gnosisFallbackLibrary",
+        "type": "address"
+      },
+      {
+        "internalType": "address",
+        "name": "_gnosisMultisendLibrary",
+        "type": "address"
+      },
+      {
+        "internalType": "address",
```

Comment:
alot of files added through the abi-generator, it allows us to export the abis in the `index.ts` and then import them from the `@baal-contracts` package to be used anywhere in the monorepo, instead of copying them from place to place.